src/components/Skeleton/Skeleton.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import styled, { keyframes } from "styled-components";
+import { SkeletonProps } from "./types";
+
+const waves = keyframes`
+   from {
+        left: -150px;
+    }
+    to   {
+        left: 100%;
+    }
+`;
+
+const pulse = keyframes`
+  0% {
+    opacity: 1;
+  }
+  50% {
+    opacity: 0.4;
+  }
+  100% {
+    opacity: 1;
+  }
+`;
+
+const Root = styled.div<{ variant: string; width: number | undefined; height: number | undefined }>`
```

Comment:
This variant type should be `Variant` right?

Reply:
I agree the type should be Variant from the SkeletonProps.